geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/delta/HAddsDeltaInfo.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.apache.geode.redis.internal.data.delta;
+
+import static org.apache.geode.redis.internal.data.delta.DeltaType.HADDS;
+
+import java.io.DataOutput;
+import java.io.IOException;
+import java.util.ArrayList;
+
+import org.apache.geode.DataSerializer;
+import org.apache.geode.internal.InternalDataSerializer;
+
+public class HAddsDeltaInfo implements DeltaInfo {
+  private final ArrayList<byte[]> deltas;
+
+  public HAddsDeltaInfo(int size) {
+    this.deltas = new ArrayList<>(size);
+  }
+
+  public HAddsDeltaInfo(byte[] fieldName, byte[] fieldValue) {
+    this(2);
+    add(fieldName);
+    add(fieldValue);
```

Comment:
IntelliJ complains about the use of an overridable method in the constructor here, which can be fixed by either making the class `final`, making the `add()` method final, or replacing these lines with: 
```
    deltas.add(fieldName);
    deltas.add(fieldValue);
```

Reply:
How do you see an intellij complaint? I don't see any warnings. Do you have some non-default warnings enabled?
\

Reply:
This is an inspection I turned on after a comment from Kirk in one of the recent lightning talks. `Preferences -> Editor -> Inspections`, then in the middle panel find `Java -> Initialization -> Overridable method called during object construction`, and check the box to enable the inspection.

Reply:
Thanks!